src/Engine/ProtoCore/Parser/atg/Start.atg

```diff
@@ -636,9 +636,8 @@ COMPILER DesignScriptParser
.'\u1751'+'\u1760'..'\u176c'+'\u176e'..'\u1770'+'\u1780'..'\u17b3'+'\u17d7'+'\u17dc'+'\u1820'..'\u1877'+'\u1880'..'\u18a8'+'\u18aa'+'\u18b0'..'\u18f5'+'\u1900'..'\u191c'+'\u1950'..'\u196d'+'\u1970'..'\u1974'+'\u1980'..'\u19ab'+'\u19c1'..'\u19c7'+'\u1a00'..'\u1a16'+'\u1a20'..'\u1a54'+'\u1aa7'+'\u1b05'..'\u1b33'+'\u1b45'..'\u1b4b'+'\u1b83'..'\u1ba0'+'\u1bae'+'\u1baf'+'\u1bba'..'\u1be5'+'\u1c00'..'\u1c23'+'\u1c4d'..'\u1c4f'+'\u1c5a'..'\u1c7d'+'\u1ce9'..'\u1cec'+'\u1cee'..'\u1cf1'+'\u1cf5'+'\u1cf6'+'\u1d00'..'\u1dbf'+'\u1e00'..'\u1f15'+'\u1f18'..'\u1f1d'+'\u1f20'..'\u1f45'+'\u1f48'..'\u1f4d'+'\u1f50'..'\u1f57'+'\u1f59'+'\u1f5b'+'\u1f5d'+'\u1f5f'..'\u1f7d'+'\u1f80'..'\u1fb4'+'\u1fb6'..'\u1fbc'+'\u1fbe'+'\u1fc2'..'\u1fc4'+'\u1fc6'..'\u1fcc'+'\u1fd0'..'\u1fd3'+'\u1fd6'..'\u1fdb'+'\u1fe0'..'\u1fec'+'\u1ff2'..'\u1ff4'+'\u1ff6'..'\u1ffc'+'\u2071'+'\u207f'+'\u2090'..'\u209c'+'\u2102'+'\u2107'+'\u210a'..'\u2113'+'\u2115'+'\u2119'..'\u211d'+'\u2124'+'\u2126'+'\u2128'+'\u212a'..'\u212d'+'\u212f'..'\u2139'+'\u213c'..'\u213f'+'\u2145'..'\u2149'+'\u214e'+'\u2160'..'\u2188'+'\u2c00'..'\u2c2e'+'\u2c30'..'\u2c5e'+'\u2c60'..'\u2ce4'+'\u2ceb'..'\u2cee'+'\u2cf2'+'\u2cf3'+'\u2d00'..'\u2d25'+'\u2d27'+'\u2d2d'+'\u2d30'..'\u2d67'+'\u2d6f'+'\u2d80'..'\u2d96'+'\u2da0'..'\u2da6'+'\u2da8'..'\u2dae'+'\u2db0'..'\u2db6'+'\u2db8'..'\u2dbe'+'\u2dc0'..'\u2dc6'+'\u2dc8'..'\u2dce'+'\u2dd0'..'\u2dd6'+'\u2dd8'..'\u2dde'+'\u2e2f'+'\u3005'..'\u3007'+'\u3021'..'\u3029'+'\u3031'..'\u3035'+'\u3038'..'\u303c'+'\u3041'..'\u3096'+'\u309d'..'\u309f'+'\u30a1'..'\u30fa'+'\u30fc'..'\u30ff'+'\u3105'..'\u312d'+'\u3131'..'\u318e'+'\u31a0'..'\u31ba'+'\u31f0'..'\u31ff'+'\u3400'..'\u4db5'+'\u4e00'..'\u9fcc'+'\ua000'..'\ua48c'+'\ua4d0'..'\ua4fd'+'\ua500'..'\ua60c'+'\ua610'..'\ua61f'+'\ua62a'+'\ua62b'+'\ua640'..'\ua66e'+'\ua67f'..'\ua697'+'\ua6a0'..'\ua6ef'+'\ua717'..'\ua71f'+'\ua722'..'\ua788'+'\ua78b'..'\ua78e'+'\ua790'..'\ua793'+'\ua7a0'..'\ua7aa'+'\ua7f8'..'\ua801'+'\ua803'..'\ua805'+'\ua807'..'\ua80a'+'\ua80c'..'\ua822'+'\ua840'..'\ua873'+'\ua882'..'\ua8b3'+'\ua8f2'..'\ua8f7'+'\ua8fb'+'\ua90a'..'\ua925'+'\ua930'..'\ua946'+'\ua960'..'\ua97c'+'\ua984'..'\ua9b2'+'\ua9cf'+'\uaa00'..'\uaa28'+'\uaa40'..'\uaa42'+'\uaa44'..'\uaa4b'+'\uaa60'..'\uaa76'+'\uaa7a'+'\uaa80'..'\uaaaf'+'\uaab1'+'\uaab5'+'\uaab6'+'\uaab9'..'\uaabd'+'\uaac0'+'\uaac2'+'\uaadb'..'\uaadd'+'\uaae0'..'\uaaea'+'\uaaf2'..'\uaaf4'+'\uab01'..'\uab06'+'\uab09'..'\uab0e'+'\uab11'..'\uab16'+'\uab20'..'\uab26'+'\uab28'..'\uab2e'+'\uabc0'..'\uabe2'+'\uac00'..'\ud7a3'+'\ud7b0'..'\ud7c6'+'\ud7cb'..'\ud7fb'+'\uf900'..'\ufa6d'+'\ufa70'..'\ufad9'+'\ufb00'..'\ufb06'+'\ufb13'..'\ufb17'+'\ufb1d'+'\ufb1f'..'\ufb28'+'\ufb2a'..'\ufb36'+'\ufb38'..'\ufb3c'+'\ufb3e'+'\ufb40'+'\ufb41'+'\ufb43'+'\ufb44'+'\ufb46'..'\ufbb1'+'\ufbd3'..'\ufd3d'+'\ufd50'..'\ufd8f'+'\ufd92'..'\ufdc7'+'\ufdf0'..'\ufdfb'+'\ufe70'..'\ufe74'+'\ufe76'..'\ufefc'+'\uff21'..'\uff3a'+'\uff41'..'\uff5a'+'\uff66'..'\uffbe'+'\uffc2'..'\uffc7'+'\uffca'..'\uffcf'+'\uffd2'..'\uffd7'+'\uffda'..'\uffdc'.
```

Comment:
Yep, unicode character could be found from http://www.unicode.org/Public/6.2.0/ucd/UnicodeData.txt